.github/workflows/pr_tests.yml

```diff
@@ -142,19 +142,12 @@ jobs:
         run: |
           pytest -m 'fast or slow' --cov syft --cov-fail-under 80 -n auto
 
-      - name: Install supported library packages Linux and MacOS - Ignore Errors
+      - name: Install supported library packages - Ignore Errors
         if: startsWith(runner.os, 'Windows') != true
         run: |
-          cat requirements/supported_libs.txt | xargs -I {} -n 1 pip install {} --default-timeout=60 || true
-        timeout-minutes: 10
-
-      - name: Install supported library packages Windows - Ignore Errors
-        if: startsWith(runner.os, 'Windows')
-        run: |
-          foreach ($i in cat requirements/supported_libs.txt) {
-            pip install $i --default-timeout=60
-          }
+          pip install requirements/supported_libs.txt
         timeout-minutes: 10
+        continue-on-error: true
```

Comment:
The problem I had was that if you pip install a .txt file and one of the packages fails it aborts the whole install. I could not find another way to ignore individual failures except to `xargs`.

Reply:
ok, now I understand what you mean, but do you know what package fails on what os?
what we did was filtering od adjustment based on OS / versions...